src/tests/posix/pthread/pthread_kill_test.c

```diff
@@ -0,0 +1,109 @@
+/**
+ * @file
+ * @brief test for pthread_kill
+ *
+ * @author  XU Chenglin
+ * @date    09/03/2020
+ */
+
+#include <pthread.h>
+#include <stdio.h>
+#include <string.h>
+#include <unistd.h>
+#include <signal.h>
+#include <embox/test.h>
+
+EMBOX_TEST_SUITE("posix/pthread_kill");
+
+void sighand(int signo) {
+    test_assert_not_zero(signo);
+    return;
+}
+
+void *thread_test(void *parm) {
+    int rc = sleep(3);
+    test_assert_not_zero(rc);
+    return NULL;
+}
+
+void *thread_ref(void *parm) {
+    int rc = sleep(3);
+    test_assert_zero(rc);
+    return NULL;
+}
+
+void *thread_immediate_return(void *parm) {
+    return NULL;
+}
+
+TEST_CASE("Send signal to a running thread") {
+    int rc;
+    struct sigaction actions;
+    pthread_t thread_normal, thread_blind;
+
+    sigemptyset(&actions.sa_mask);
+    actions.sa_flags = 0;
+    actions.sa_handler = sighand;
+    rc = sigaction(SIGALRM,&actions,NULL);
```

Comment:
Please use spases
(SIGALRM,&actions,NULL)->(SIGALRM, &actions, NULL)

Reply:
Thank you so much for your patience. I will be more careful.